ios/Mobile.xcodeproj/project.pbxproj

```diff
@@ -3863,7 +3863,6 @@
 					"DISABLE_DYNLOADING=1",
 					"ENABLE_DEBUG=1",
 					"COOLWSD_CONFIGDIR='\".\"'",
-					"COOLWSD_LOGLEVEL='\"trace\"'",
```

Comment:
If this is truly unused, remove it also from android/lib/src/main/cpp/CMakeLists.txt.in?

I'm but exactly sure how this works, but seems DefAppConfig in common/ConfigUtil.cpp still uses COOLWSD_LOGLEVEL, which seems to be outside any "if not on mobile" ifdefs, so I guess this was then indeed redundant and you're right.